dvc/tree/http.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, repo, config):
         self.ask_password = config.get("ask_password", False)
         self.headers = {}
         self.ssl_verify = config.get("ssl_verify", True)
+        self.http_method = config.get("http_method", "POST")
```

Comment:
Btw, are you using dvc through API? For CLI we also need to add this to https://github.com/iterative/dvc/blob/master/dvc/config.py#L121

Reply:
Also, would changing the name from `http_method` to `method` make sense to you?

Reply:
yep! fixed